API/models-json/Open-Ethics-Data-Passport.json

```diff
@@ -0,0 +1,46 @@
+{
+  "title": "Open Ethics Data Passport",
+  "type": "object",
+  "description": "OEDP Model",
+  "properties": {
+    "id": {
+      "type": "string",
+      "format": "uuid"
+    },
+    "datasets": {
```

Comment:
@abhi12ravi it seems that the schema still follows the old structure, we'd need to advance it to the new one. E.g. a passport of the product may employ several models, which in turn may employ several datasets.

In the previous version we considered unimodular AI products but, as you know, today many products may embed several models simultaneously.

In addition, we may require to describe models alone when there's absolutely no info on the datasets. 

From the issue description
> Follow the newest hierarchy described here https://openethics.ai/oedp/ (not yet in the YAML) to incorporate the idea that there could be multiple models supporting a product

Reply:
e.g. we'd need to have a new branch element called "models" which will be an array of Models defined by a new schema called "Model.JSON" and a model schema in turn will contain datasets as optional

Reply:
@nikiluk I'm unable to follow, could we have a call around this?